python/__init__.py

```diff
@@ -69,3 +70,15 @@ def setupenv():
 
 
 from qgis.PyQt import QtCore
+
+# monkey patching custom widgets in case we running on a local install
+# should fix import errors ModuleNotFoundError: No module named qgsfilewidget
+try:
+    import qgis.gui
+    widget_list = dir(qgis.gui)
+    widget_list.remove('QgsScrollArea')
+    for widget in widget_list:
+        if widget.startswith('Qgs'):
+            exec("sys.modules['{module}'] = qgis.gui".format(module=widget.lower()))
```

Comment:
why using exec?
does it work with `sys.modules[widget.lower()] = qgis.gui`?

Reply:
kind of a left-over from another try.
indeed will change it